terraform/stacks/tooling/stack.tf

```diff
@@ -313,3 +320,18 @@ resource "aws_iam_policy_attachment" "concourse_worker" {
     "${module.concourse_worker_role.role_name}"
   ]
 }
+
+module "operators" {
+  source = "../../modules/operators"
+}
+
+resource "aws_iam_policy_attachment" "admin" {
+  name = "admin"
+  policy_arn = "arn:aws-us-gov:iam::aws:policy/AdministratorAccess"
```

Comment:
This is ok to be here twice (other in external stack) because external runs in the other non-govcloud aws account right?